### PR TITLE
Support packages (PS-896)

### PR DESCRIPTION
**Pull request by JakubMatejka**

### CODE REVIEW

File: src/Sandbox.php [hunk 1]
```diff
@@ -85,6 +86,9 @@ public function __construct(?array $sandbox = null)
         if (!empty($sandbox['autosaveTokenId'])) {
             $this->setAutosaveTokenId((string) $sandbox['autosaveTokenId']);
         }
+        if (!empty($sandbox['packages'])) {
+            $this->setImageVersion($sandbox['packages']);
```
Comment (pivnicek) [inline src/Sandbox.php:90]: oh, this looks wrong :-)

**Review by pivnicek — CHANGES_REQUESTED** [review]
tiny chyba:)

File: src/Sandbox.php (hunk 1 above)
Comment (JakubMatejka) [reply src/Sandbox.php:90]: omg 🙈

**Review by pivnicek — APPROVED** [review]
👍

**Pull request merged**